travel-event-host-app/src/app/clients/event/event-client.ts

```diff
@@ -1,28 +1,24 @@
-import CategoriesType from '@/components/searchSection/types';
+import Category from '@/lib/category';
 import Event from '@/models/event';
 
 export async function getEventsBySearchQuery(
   keyword: string,
-  categories: CategoriesType,
+  categories: Category[],
```

Comment:
Instead of passing the entire categories: boolean object, we can simply pass a `string[]`